### ISSUE TITLE
master broken, wrong license families

### ISSUE THREAD

**Issue by ilanschnell** [conda/conda-build#1504]
Using master 6f13867d4208040b6ddfa17b1eb2c102b032b7de, I get:
```
about/license_family 'Public-Domain' not allowed. Allowed families are
AGPL, APACHE, BSD, GPL, GPL2, GPL3, LGPL, MIT, NONE, OTHER,
PROPRIETARY, PSF, and PUBLICDOMAIN.
```
The recipe has:
```
about:
  license_family: Public-Domain
```
which is completely valid.  The license families are defined here:
https://github.com/ContinuumIO/anaconda-verify/blob/master/anaconda_verify/const.py#L1

This list has been used for many years now, and I don't see the point of changing punctuation, capitalization or anything else about the list.

**Comment by msarahan** [comment]
Fixed by #1503 - closing, but please reopen if there are further issues.

**Comment by ilanschnell** [comment]
Thanks, #1503 fixed the problem.
